docs/graphql/core/schema/tables.rst

```diff
@@ -44,7 +44,7 @@ Let's say we want to create two simple tables for an article/author schema:
 
   .. tab:: Console
 
-    Open the Hasura console and head to the ``Data`` tab and click the ``Create Table`` button to open up an interface to
+    Open the Hasura console and head to the ``Data`` tab and click the ``Add Table`` button on the left side bar to open up an interface to
```

Comment:
This is a bit confusing because we talk about the `Add table` button on the left side bar and then the other `Add table` button is highlighted in the below picture. Can we fix this?